src/logic/wallets/getWeb3.js

```diff
@@ -145,10 +94,10 @@ const getNetworkIdFrom = async web3Provider => {
   return networkId
 }
 
-export const getProviderInfo: Function = async (web3Provider): Promise<ProviderProps> => {
+export const getProviderInfo: Function = async (web3Provider, providerName?: string): Promise<ProviderProps> => {
   web3 = new Web3(web3Provider)
 
-  const name = getProviderName(web3)
+  const name = providerName
```

Comment:
if name is not provider what would it show? It's safer if it'd be like this:
```
const name = providerName || 'Wallet'
```

or set a default value for function argument